### ISSUE TITLE
Need to remove deprecated  pd.rolling_apply 

### ISSUE THREAD

**Issue by luca-s** [quantopian/alphalens#166]
Need to fix this when we feel like we want to increase the minimum pandas version:

```
/usr/local/lib/python2.7/dist-packages/alphalens/plotting.py:727: FutureWarning: pd.rolling_apply is deprecated for Series and will be removed in a future version, replace with 
	Series.rolling(center=False,min_periods=1,window=3).apply(args=<tuple>,func=<function>,kwargs=<dict>)
```

**Comment by georgh0021** [comment]
There are two places where we use `pd.rolling_apply`, both in `plotting.py`: [here](https://github.com/quantopian/alphalens/blob/c8ba9b8363595f1296ea9fc78a5b0443d7cceb29/alphalens/plotting.py#L726) and [here](https://github.com/quantopian/alphalens/blob/c8ba9b8363595f1296ea9fc78a5b0443d7cceb29/alphalens/plotting.py#L766).

Conveniently, pandas 0.20.3 has [a new `Rolling` class with an `apply` attribute!](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.core.window.Rolling.apply.html) This looks like it will be a fairly painless 2-line fix.

**Comment by georgh0021** [comment]
Just remembered: we need to be careful about supporting the right versions of pandas. [This issue on pyfolio](https://github.com/quantopian/pyfolio/issues/407) is what happens when zipline, pyfolio and alphalens support different versions. Best to verify the versions before opening a PR for this,

**Comment by luca-s** [comment]
My poor comment "when we feel like we want to increase the minimum pandas version" meant exactly that. This issue was just a reminder that we have to get rid of the deprecation warning eventually, but I see that it creates noise, so I'll close it now.

**Comment by georgh0021** [comment]
@luca-s no, it is a valid point and not noise at all! It's better to keep this issue open to remind us about the versioning issue.

Firstly, pandas will deprecate `rolling_apply` sometime, so we will have to deal with this issue eventually: better sooner than later.

Secondly, this issue underlines a problem across alphalens, zipline and pyfolio: their dependencies should be coordinated in lockstep, otherwise it ruins the productivity of anyone trying to use the libraries together (as they should).

cc'ing @richafrank (tldr, pyfolio supports pandas >0.20, zipline supports pandas <0.18.0, alphalens depends on deprecated pandas functions from 0.18.0)

**Comment by luca-s** [comment]
In case you are increasing the minimum pandas version for Alphalens I would suggest >= 0.20 because there were few bugs in pandas.cut and pandas.qcut in earlier versions.

Anyway, because Quantopian supports pandas 0.18 at the moment, I don't see this happening right now.

**Comment by luca-s** [comment]
fixed with PR #195
